app/src/main/res/layout/fragment_q_r_generator.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".view.QR.QRGeneratorFragment">
+
+    <ImageView
+        android:layout_width="200dp"
```

Comment:
Have you tried the layout on a big screen ? I feel like the fixed size might create a really small QR code on big screens. Or maybe this is needed, I don't know...

Reply:
It is large enough, if I make it bigger it might be "flou"